### PR TITLE
group children's tags even there's no definition for tags

### PR DESCRIPTION
**Pull request by hellosnow**
@vwxyzh @ansyral @superyyrrzz @qinezh @chenkennt

### CODE REVIEW

File: src/docfx.website.themes/common/RestApi.common.js [hunk 1]
```diff
@@ -22,6 +22,29 @@ exports.transform = function (model) {
...
+                if (child.tags && child.tags.length > 0) {
+                    for (var k = 0; k < child.tags.length; k++) {
+                        // for each tag in child, add unique tag string into childTags
+                        if (childTags.indexOf(child.tags[k]) == -1) {
+                            childTags.push(child.tags[k]);
+                        }
+                    }
+                }
+            }
+            // sort alphabetically
```
Comment (hellosnow) [inline src/docfx.website.themes/common/RestApi.common.js]: remove spaces

**Comment by superyyrrzz** [comment]
:+1:

**Comment by vwxyzh** [comment]
👍

File: src/docfx.website.themes/common/RestApi.common.js [hunk 2]
```diff
@@ -22,6 +22,29 @@ exports.transform = function (model) {
             resolveAllOf(child);
             transformReference(child);
         };
+        if (!model.tags || model.tags.length == 0) {
```
Comment (qinezh) [inline src/docfx.website.themes/common/RestApi.common.js]: minor: always use '===' in javascript

**Comment by qinezh** [comment]
:shipit:

File: src/docfx.website.themes/common/RestApi.common.js (hunk 2 above)
Comment (hellosnow) [inline src/docfx.website.themes/common/RestApi.common.js]: for array.length, which return value is number, I think use == is ok here.

---
In reply to: [68890026](https://github.com/dotnet/docfx/pull/442#discussion_r68890026) [](ancestors = 68890026)

Comment (hellosnow) [inline src/docfx.website.themes/common/RestApi.common.js]: according to [http://www.w3schools.com/js/js_best_practices.asp](http://www.w3schools.com/js/js_best_practices.asp), should avoid ==

---
In reply to: [68890788](https://github.com/dotnet/docfx/pull/442#discussion_r68890788) [](ancestors = 68890788,68890026)

**Pull request merged**